lib/geocoder/providers/open_street_maps.ex

```diff
@@ -107,6 +107,7 @@ defmodule Geocoder.Providers.OpenStreetMaps do
   #   "osm_id" => "45352282", "osm_type" => "way", "place_id" => "70350383"}
   @map %{
     "house_number" => :street_number,
+    "suburb" => :county,
```

Comment:
We should note this in code:

```suggestion
     # Australia suburbs are used instead of counties: https://github.com/knrz/geocoder/pull/71
    "suburb" => :county,
```